bitnami/redis/Chart.yaml

```
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 12.7.7
+version: 12.7.8
```

As it's a new feature more than a bug fix, I would bump the minor version instead
```suggestion
version: 12.8.0
```